src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.CodeAnalysis.Sarif.VersionOne;
+
+namespace Microsoft.CodeAnalysis.Sarif.Visitors
+{
+    public static class SarifTransformerUtilities
+    {
+        #region Text MIME types
```

Comment:
I've seen these types defined elsewhere. Don't duplicate them.

Reply:
Oh hah I see you just moved them. GOod.

Reply:
This is a move